report/firefox_code_coverage/codecoverage.py

```diff
@@ -39,8 +39,7 @@ def is_taskcluster_loaner():
 
 def get_task(branch, revision):
     task = get_json(
-        "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.%s.revision.%s.firefox.linux64-ccov-opt"
-        % (branch, revision)
+        f"https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.{branch}.revision.{revision}.firefox.decision"
```

Comment:
I had a hard time parsing that diff and noticing the `decision` change at the end. :coffee: